src_py/apiServer/networkComponents.py

```diff
@@ -66,10 +66,10 @@ def __init__(self, arch_json):
         # Checking is we are running an AEC in the current architecture:
         model = self.jsonData['workers'][0]['modelType']
 
-        if (model == '7'):
-            self.aec = 1
-        else:
-            self.aec = 0
+        # if (model == '7'):
```

Comment:
Get rid of that code if not needed 
What is the magic number 7? 
I don't like it 

If it might be needed add some comment here what this block does

Reply:
Done